src/Resources/app/storefront/src/js/filter-category-select.plugin.js

```diff
@@ -36,22 +51,18 @@ export default class FilterCategorySelectPlugin extends FilterBasePlugin {
      * @public
      */
     getValues() {
-        const activeCheckboxes = this.getSelected();
-
-        let selection = [];
 
-        if (activeCheckboxes) {
-            selection.push(activeCheckboxes[0].value);
-        } else {
-            selection = [];
+        const selectedCategories = this.getSelected();
+        const selection = [];
```

Comment:
Rename this to `selectedCategories` or `selectedFilterValues`, as `selection` is a bit to general.

```suggestion
        const selectedCategories = [];
```

As you already used `selectedCategories` in other places aswell, it probably makes sense to use this.